config/env/defaults.js

```diff
@@ -34,19 +34,18 @@ async function get() {
         ws: ws,
         db: db,
         queueDb: queueDb,
-        giftAmount: Number(process.env.GIFT_AMOUNT) || 0.3,
-        refundThreshold: Number(process.env.REFUND_THRESHOLD) || 0.1,
-        contractCreateGasAmount: Number(process.env.CONTRACT_CREATE_GAS_AMOUNT) || 50000,
-        contractCallGasAmount: Number(process.env.CONTRACT_CALL_GAS_AMOUNT) || 10000,
+        giftAmount: Number(valueOrDefault(process.env.GIFT_AMOUNT, 0.3)),
+        refundThreshold: Number(valueOrDefault(process.env.REFUND_THRESHOLD, 0.1)),
+        contractCreateGasAmount: Number(valueOrDefault(process.env.CONTRACT_CREATE_GAS_AMOUNT, 50000)),
```

Comment:
fyi, some languages like JS have an option to delimit digits, which can provide better readability. 
e.g. it's correct to write `50_000` and it would be set as `50000`. It's absolutely fine, but maybe it can help you in the future. just maybe :P